theme.qml

```diff
@@ -137,10 +137,18 @@ FocusScope {
     property var allCollections: {
         const collections = api.collections.toVarArray();
 
-        collections.unshift({'name': 'Favorites', 'shortName': 'favorites', 'games': allFavorites});
-        collections.unshift({'name': 'Last Played', 'shortName': 'recents', 'games': filterLastPlayed});
-        collections.unshift({'name': 'All Games', 'shortName': 'allgames', 'games': api.allGames});
-
+		if (settings.get('showFavorites')) {
+			collections.unshift({'name': 'Favorites', 'shortName': 'favorites', 'games': allFavorites});
+        }
+		
+		if (settings.get('showRecents')) {
+			collections.unshift({'name': 'Last Played', 'shortName': 'recents', 'games': filterLastPlayed});
+        }
+		
+		if (settings.get('showAllGames')) {
+			collections.unshift({'name': 'All Games', 'shortName': 'allgames', 'games': api.allGames});
+		}
+		
```

Comment:
I actually tested through these pretty extensively, and spent a bunch of time tinkering with the config code in general to see if I could resolve the strange behaviors unrelated to these specific configs.

I've noticed on my AYN Odin Pro that I routinely see an issue where the theme config JSON file ends up in some bizarre corrupted state that causes it to fail to load, leading to all configs being reset to default. In addition, sometimes when using the config view the boxes will not check/uncheck despite the backing values themselves changing. Very odd. I suspect it may have something to do with a bug in the `api.memory` space in the Pegasus API.

With that in mind, as long as the config file isn't borked these configs always work on startup. They SHOULD also work when you change the values without needing to restart Pegasus, but that only seems to happen about half of the time, due to the aforementioned config behavior oddities.

Reply:
I've had some other people mentioning that their settings get reset in the discord but I haven't been able to reproduce it at all myself (let alone reliably). haven't heard if any other themes have a similar issue. it might be just an android thing. thanks for testing!